src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -305,15 +308,18 @@ private static void SetProviderConfiguration(string strConfig, EventPipeConfigur
                 throw new ArgumentNullException(nameof(strConfig));
             }
 
-            // String must be of the form "providerName:keywords:level:parameter,providerName:keywords:level:parameter..."
-            // where parameter is of the form: "key=value;key=value..."
+            // Provider format: "(GUID|KnownProviderName)[:Flags[:Level][:EventCouter]]"
+            // where EventCouters are of the form: "[key1=value1][;key2=value2]"
```

Comment:
Don't call it EventCounter, call it KeyValueArgs, because any EventSource can use the parameters for whatever they want (including but not exclusive to EventCounters.